frontend/src/routes/CriticalQuestions/index.js

```diff
@@ -510,39 +526,27 @@ function CriticalQuestions(props) {
           </FormControl>
         </div>
         {closeContactsSection()}
-        {travelsSection()}
+        {/*travelsSection()*/}
         <div style={{ height: "30px" }} ref={pageBottomRef}></div>
       </div>
 
-      {contacts.filter((contact) => contact.email).length !== 0 &&
-        formValues[fields.CITY.key] &&
-        formValues[fields.CITY.key].length &&
-        formValues[fields.STATE.key] &&
-        formValues[fields.STATE.key].length &&
-        formValues[fields.COUNTRY.key] &&
-        formValues[fields.COUNTRY.key].length && (
-          <AlertDialog
-            label={
-              <Fab
-                style={{ background: "#EA2027" }}
-                aria-label="Go to next page"
-                size="medium"
-                className="fab next-btn"
-              >
-                <ArrowRightIcon />
-              </Fab>
-            }
-            text={contactNoticeText}
-            submit={handleSubmit}
-          />
-        )}
-      {(contacts.filter((contact) => contact.email).length === 0 ||
-        !formValues[fields.CITY.key] ||
-        !formValues[fields.CITY.key].length ||
-        !formValues[fields.STATE.key] ||
-        !formValues[fields.STATE.key].length ||
-        !formValues[fields.COUNTRY.key] ||
-        !formValues[fields.COUNTRY.key].length) && (
```

Comment:
Still need these lines as the user is sick and does not put the state and county, we should remind them to fill those required info first.

Reply:
I don't think these lines actually changes anything because the submitHistory takes care of the restriction. I tried leaving the [fields.CITY.key].length and removed the required property from story.js and was able to move forward.

Reply:
If we delete these lines, the email sending notification will pop out first instead of reminding the user to fill the required fields if the user did not do so.